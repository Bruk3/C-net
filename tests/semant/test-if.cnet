int main(int a, string argv) {
    int b = 4;
    if (a > b) {
        stdout.println("a is greater than b");
    }
    else if (a == b) {
        stdout.println("a is equal to b");
    }
    else if (a <= b ){
        stdout.println("a is less than or equal to b");
    }
    else {
        stdout.println("b is greater than a");
    }
}