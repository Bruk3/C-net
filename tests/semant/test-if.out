int main(string[] argv)
{
int a = (int : 5);
int b = (int : 4);
{
if ((int : (int : a) > (int : b)))
{
(int : writeln((file : cnet_stdout), (string : "a is greater than b")));
}
else if ((int : (int : a) == (int : b)))
{
(int : writeln((file : cnet_stdout), (string : "a is equal to b")));
}
else if ((int : (int : a) <= (int : b)))
{
(int : writeln((file : cnet_stdout), (string : "a is less than or equal to b")));
}
else if ((int : 1))
{
(int : writeln((file : cnet_stdout), (string : "b is greater than a")));
}
}
return (int : 0);
}

